src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/viewmodels/DocumentsViewModel.kt

```diff
@@ -16,9 +16,18 @@ data class DocumentViewModel(val displayName: String,
     {
         fun build(doc: Document, appConfig: Config = AppConfig()): DocumentViewModel?
         {
+            val url = if (doc.external)
+            {
+                doc.displayName
```

Comment:
I guess when we actually start making us of custom displayNames we'll have to pass name and displayName to this method, but for now displayName is always = name.

Reply:
Right, yeah that is a bit confusing. I can see it's because we don't actually have name on the Document class. Maybe put a comment here to that effect, as it definitely seems a bit weird that the url should be the display name!